Library/Formula/camlp5.rb

```diff
@@ -13,20 +13,41 @@ class Camlp5 < Formula
 
   deprecated_option "strict" => "with-strict"
   option "with-strict", "Compile in strict mode"
+  option "with-doc", "Install the html and info documentation"
+  option "with-tex", "Install the pdf, ps, and tex documentation"
 
   depends_on "ocaml"
+  depends_on "gnu-sed" => :build if build.with?("doc") || build.with?("tex")
+  depends_on :tex => [:build, :optional]
 
   def install
-    if build.with? "strict"
-      strictness = "-strict"
-    else
-      strictness = "-transitional"
+    args = ["--prefix", prefix, "--mandir", man]
+    args << "--transitional" if build.without? "strict"
+
+    if build.with?("tex") && (ps2pdf = which("ps2pdf")).nil?
+      saved_path = ENV["PATH"]
+      ENV.append_path "PATH", "/usr/local/bin" # known install location by MacTeX
+      odie "Couldn't find ps2pdf" if (ps2pdf = which("ps2pdf")).nil?
+      ENV["PATH"] = saved_path
```

Comment:
A fancier way is to check the package database.
```
bash-3.2$ pkgutil --pkgs |grep ghost
org.tug.mactex.ghostscript9.16
bash-3.2$ pkgutil --lsbom org.tug.mactex.ghostscript9.16|grep -E 'ps2pdf$'
./bin/ps2pdf
bash-3.2$ 
```
(it's ./bin since the pkg specifies the prefix as /usr/local elsewhere)